packages/react/test/specs/utils/__snapshots__/felaRenderer-test.tsx.snap

```diff
@@ -124,7 +124,7 @@ exports[`felaRenderer marginLeft is rendered into marginLeft due to RTL with \`n
 
 
 <div className=ui-provider__box dir=rtl>
-  <span className=ui-text a dir=auto>
+  <span dir=auto className=ui-text a>
```

Comment:
I wonder why these are changed :\

Reply:
Small issue, I fixed order of props in `Text` and reverted this change 👍